### ISSUE TITLE
Error: no artifacts found (but rerun is ok)

### ISSUE THREAD

**Issue by llicour** [dawidd6/action-download-artifact#43]
I have the same issue #42 

My used case is using fountainhead/action-wait-for-check@v1.0.0 action before using action-download-artifact
I think this is something like a timing issue in github. This is not reproductible on each execution... I had to put a sleep step of 10s before starting downloading artifact (hoping it is ok)

Perhaps a better way would be to manage a retry with a backoff in this action...

**Comment by dawidd6** [comment]
Consider making a PR if this feature would be useful for you.

**Comment by Jonatino** [comment]
Same issue

**Comment by dawidd6** [comment]
Any examples of failing workflows? Files, links, anything?

**Comment by ynouri** [comment]
Not super helpful comment, but I am running into the same issue in a private repo. As mentioned above, the root cause is probably on Github side (small latency before artifacts are available through the API?)

One nice-to-have feature for `dawidd6/action-download-artifact` could be an additional input `sleep` in seconds for example. Would remove the need for an additional sleep step in the job.

Anyway, thanks this awesome action! It is super useful
